docs/docs/ref-05-events.md

```diff
@@ -31,7 +31,7 @@ string type
 
 > Note:
 >
-> As of v0.12, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
```

Comment:
I'm fairly certain support for this was already removed in 0.13 and it's no longer "in a future release". I think we should just change this to `0.13` and call it a day. cc @spicyj